src/components/ThirdPartyScripts/vtex.tsx

```diff
@@ -1,10 +1,7 @@
-import React from 'react'
-
 function VTEX() {
   return (
     <>
       <script
-        key="vtexrc.js-init"
```

Comment:
Why did you remove them? I know those scripts won't be added or removed, but it helps React understand who those tags are.

Reply:
why?